core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -261,7 +261,7 @@ public long acceptWay( OSMWay way )
             return 0;
 
         // use the way if it is tagged for bikes
-        if (way.hasTag("bicycle", intendedValues))
+        if (way.hasTag("bicycle", intendedValues) || way.hasTag("bicycle", "dismount"))
```

Comment:
I had this first. It is not a good idea for two reasons:
1) The code in `getSpeed` in BikeCommonFlagEncoder.java would require at least one more 'if'
2) It is not logical. A way with `bicycle=dismount` is not really intended for bicycle usage such as the other values `designated` or `official`.